scene/gui/scroll_container.cpp

```diff
@@ -542,13 +536,10 @@ PackedStringArray ScrollContainer::get_configuration_warnings() const {
 	int found = 0;
 
 	for (int i = 0; i < get_child_count(); i++) {
-		Control *c = Object::cast_to<Control>(get_child(i));
+		Control *c = as_sortable_control(get_child(i));
```

Comment:
The warning message below could be changed to mention it's supposed to work _"with a single **visible** child"_.

Reply:
We could add an argument for the visibility check to make it fully match the original code

Reply:
The warning does not mention `top_level` either.
Changing this to `is_visible_in_tree()` is only a problem if the ScrollContainer itself is invisible. The warning will not appear in this case. I don't think it's worth handling.